### PR TITLE
[do not merge] dimensions/measures/entities support meta tag

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds a new feature where dimensions/measures/entities  supports `meta` config tag.

currently waiting for eng update on this as still working through

[notion docs project](https://www.notion.so/dbtlabs/Semantic-Layer-MF-meta-and-tag-updates-12ebb38ebda780fdb97ec44de2df036a?pvs=4)

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-add-meta-tag-dbt-labs.vercel.app/docs/build/dimensions
- https://docs-getdbt-com-git-add-meta-tag-dbt-labs.vercel.app/docs/build/entities
- https://docs-getdbt-com-git-add-meta-tag-dbt-labs.vercel.app/docs/build/measures
- https://docs-getdbt-com-git-add-meta-tag-dbt-labs.vercel.app/docs/build/simple
- https://docs-getdbt-com-git-add-meta-tag-dbt-labs.vercel.app/docs/dbt-versions/release-notes
- https://docs-getdbt-com-git-add-meta-tag-dbt-labs.vercel.app/reference/resource-configs/meta

### CODE REVIEW

File: website/docs/docs/build/dimensions.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ All dimensions require a `name`, `type`, and can optionally include an `expr` pa
 | `description` | A clear description of the dimension. | Optional | String |  
 | `expr` | Defines the underlying column or SQL query for a dimension. If no `expr` is specified, MetricFlow will use the column with the same name as the group. You can use the column name itself to input a SQL expression. | Optional | String |
 | `label` | Defines the display value in downstream tools. Accepts plain text, spaces, and quotes (such as `orders_total` or `"orders_total"`).  | Optional | String |
+| [`meta`](/reference/resource-configs/meta) |  Set metadata for a resource and organize resources. Accepts plain text, spaces, and quotes. | Optional | {<dictionary>} |
```
Comment (mirnawong1) [inline website/docs/docs/build/dimensions.md]:
```suggestion
| [`meta`](/reference/resource-configs/meta) |  Set metadata for a resource and organize resources. Accepts plain text, spaces, and quotes. | Optional | <dictionary> | 
```

**Review by runleonarun — APPROVED** [review]
This looks great @mirnawong1 My only question is can you configure meta with _any_ metadata? Or are there set values for it? It seems like it can be anything but we don't explicitly say that on the meta page.

**Comment by mirnawong1** [comment]
> This looks great @mirnawong1 My only question is can you configure meta with _any_ metadata? Or are there set values for it? It seems like it can be anything but we don't explicitly say that on the meta page.

hey @mirnawong1 , yes `meta` includes any custom key-value pair, as long as it's valid yaml. so you can use it for owner, sla, etc anything related to your use case. so i'[ve used `favorite_food` here and it was visible in th manifest.json file:
Screenshot 2024-12-24 at 10 36 45

**Comment by mirnawong1** [comment]
thanks @runleonarun , added a clarification about the meta acceptance here: 

Screenshot 2024-12-24 at 10 42 24

going to merge this now!

**Pull request merged**